modules/nw-create-load-balancer-service.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * ingress/getting-traffic-cluster.adoc
+
+[id="nw-create-load-balancer-service_{context}"]
+= Creating a load balancer service
+
+Use the following procedure to create a load balancer service.
+
+.Prerequisites
+
+* Make sure you have created your project and service.
+
+.Procedure
+
+To create a load balancer service:
+
+. Log into {product-title}.
+
+. Load the project where the service you want to expose is located.
++
+----
+$ oc project project1
+----
+
+. Open a text file on the master node and paste the following text, editing the
+file as needed:
++
+.Sample load balancer configuration file
+====
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: egress-2 <1>
+spec:
+  ports:
+  - name: db
+    port: 3306 <2>
+  loadBalancerIP:
+  type: LoadBalancer <3>
+  selector:
+    name: mysql <4>
+----
+
+<1> Enter a descriptive name for the load balancer service.
+<2> Enter the same port that the service you want to expose is listening on.
+<3> Enter `loadbalancer` as the type.
+<4> Enter the name of the service.
+====
+
+. Save and exit the file.
+
+. Run the following command to create the service:
++
+----
+oc create -f <file-name>
+----
++
+For example:
++
+----
+oc create -f mysql-lb.yaml
+----
+
+. Execute the following command to view the new service:
++
+----
+oc get svc
+NAME              CLUSTER-IP       EXTERNAL-IP                   PORT(S)                   AGE
+egress-2          172.30.236.167   172.29.121.74,172.29.121.74   3306/TCP                  6s
+----
```

Comment:
The column `TYPE` is missing in above output, and here is an example:  
```
$ oc get svc -n openshift-ingress
NAME                      TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)                      AGE
router-default            LoadBalancer   172.30.16.119   52.230.228.163   80:30745/TCP,443:32561/TCP   2d6h
router-internal-default   ClusterIP      172.30.101.15   <none>           80/TCP,443/TCP,1936/TCP      2d6h
```
The first line is LoadBalaner service, and second one is normal `ClusterIP` service.